lib/mongoose_push_web/controllers/api_v1_notification_controller.ex

```diff
@@ -5,6 +5,10 @@ defmodule MongoosePushWeb.APIv1.NotificationController do
 
   use MongoosePushWeb.Schemas
 
+  plug(OpenApiSpex.Plug.CastAndValidate)
+
+  def mongoose_push, do: Application.get_env(:mongoose_push, :mongoose_push)
```

Comment:
That's a weird function name, how about `backend_module`?

Reply:
Also, let's use `get_env!` or something similar, so that we crash when this setting is not set, instead crashing in weird `undef` reason.

Reply:
This function also should be moved to `MongoosePush.Application` module, where all configuration is handled. This is especially useful, as we use this in 3 modules.

Reply:
done